docker-run.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+set
+
+# git ROOT directory used to mount filesystem
+GIT_ROOT_DIRECTORY=$(git rev-parse --show-toplevel)
+GO_MODULE=$(grep -e 'module ' ${GIT_ROOT_DIRECTORY}/go.mod | sed -e 's/module //')
+WORKDIR="/home/user/go/src/${GO_MODULE}"
+# Container image
+IMAGE_NAME="che-incubator/devworkspace-build-prerequisistes"
+
+# Operator SDK
+OPERATOR_SDK_VERSION=v0.12.0
+
+init() {
+  BLUE='\033[1;34m'
+  GREEN='\033[0;32m'
+  RED='\033[0;31m'
+  NC='\033[0m'
+  BOLD='\033[1m'
+}
+
+check() {
+  if [ $# -eq 0 ]; then
+    printf "%bError: %bNo script provided. Command is $ docker-run.sh <script-to-run> [optional-arguments-of-script-to-run]\n" "${RED}" "${NC}"
+    exit 1
+  fi
+  echo "check $1"
+  if [ ! -f "$1" ]; then
```

Comment:
Should also check file is executable:
```suggestion
  if [ ! -f "$1" ] || [ ! -x "$1"]; then
```